src/go/MONIT/alert.go

```diff
@@ -576,31 +576,31 @@ func main() {
 		fmt.Println("\nEnvironments:")
 		fmt.Println("\tCONFIG_PATH:\t Config Filepath")
 		fmt.Println("\nExamples:")
-		fmt.Println("\t-Get all alerts:")
-		fmt.Println("\t alert")
-		fmt.Println("\t-Get all alerts in JSON format:")
-		fmt.Println("\t alert -json")
-		fmt.Println("\t-Get all alerts with filters (-json flag will output in JSON format if required):")
-		fmt.Println("\t  Available filters:")
-		fmt.Println("\t  service\tGGUS,SSB,dbs")
-		fmt.Println("\t  severity\tinfo,medium,high,urgent etc.")
-		fmt.Println("\t  tag\t\tcmsweb,cms,monitoring etc.")
-		fmt.Println("\n\t  Get all alerts of specific service/severity/tag. Ex GGUS/high/cms:")
-		fmt.Println("\t   alert -service=GGUS")
-		fmt.Println("\t   alert -severity=high")
-		fmt.Println("\t   alert -tag=cms")
-		fmt.Println("\n\t  Get all alerts based on multi filters. Ex service=GGUS, severity=high:")
-		fmt.Println("\t   alert -service=GGUS -severity=high")
-		fmt.Println("\t-Sort alerts based on labels. The -sort flag on top of above queries will give sorted alerts.:")
-		fmt.Println("\t  Available labels:")
-		fmt.Println("\t  severity\tSeverity Level")
-		fmt.Println("\t  starts\tStarting time of alerts")
-		fmt.Println("\t  ends\t\tEnding time of alerts")
-		fmt.Println("\t  duration\tLifetime of alerts")
-		fmt.Println("\n\t  Get all alerts of service=GGUS, severity=high sorted on alert's duration:")
-		fmt.Println("\t   alert -service=GGUS -severity=high -sort=duration")
-		fmt.Println("\n\t  Get all alerts of service=GGUS sorted on severity level:")
-		fmt.Println("\t   alert -service=GGUS -sort=severity")
+		fmt.Println("\tGet all alerts:")
+		fmt.Println("\t    alert")
+		fmt.Println("\tGet all alerts in JSON format:")
+		fmt.Println("\t    alert -json")
+		fmt.Println("\tGet all alerts with filters (-json flag will output in JSON format if required):")
+		fmt.Println("\t    Available filters:")
+		fmt.Println("\t    service\tEx GGUS,SSB,dbs,etc.")
+		fmt.Println("\t    severity\tEx info,medium,high,urgent,etc.")
+		fmt.Println("\t    tag\t\tEx cmsweb,cms,monitoring,etc.")
+		fmt.Println("\n\t    Get all alerts of specific service/severity/tag. Ex GGUS/high/cms:")
```

Comment:
I'm not sure if you need additional tag/spaces here. I would suggest that you align all "Get" messages on the same column.

Reply:
Valentin, 
The tabbed "Get" messages are sub-messages. Should I still align them in same column?

Reply:
All you gets, are just examples, do not complicate things where it is not necessary, it only leads to confusion from user side

Reply:
Okay. I have made changes.